@xen-orchestra/proxy/src/app/mixins/backups/index.js

```diff
@@ -69,6 +69,14 @@ export default class Backups {
         }
       }
     })(run)
+    run = (run =>
+      async function () {
+        const license = await app.appliance.getSelfLicense()
+        if (license === undefined || license.expires < Date.now()) {
+          throw new Error('the proxy license is not valid')
```

Comment:
Can we disable this for dev builds?

Maybe using https://github.com/martinandert/babel-plugin-transform-dev